net/HttpRequest.hpp

```diff
@@ -1723,8 +1723,9 @@ class Session final : public ProtocolHandlerInterface
         }
 
         _connected = false;
-        if (_response)
-            _response->finish();
+        if (_response) {
```

Comment:
Braces are good, but maintaining the style is even better. (The bracing style is on-new-line.)

(Btw, we use clang-format to do auto-formatting, and if you configure it to format changed lines only, it works great.)

Reply:
It does have https://clang.llvm.org/docs/ClangFormatStyleOptions.html#removebracesllvm

But it is not in .clang-format.
And clang-format is not checked by the pre-commit hook.
I would change that gladly, but I don't know anything about perl and I don't want this to change, python would have been nice or just bash.